addons/stock_transport/models/stock_picking.py

```diff
@@ -0,0 +1,51 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
```suggestion
```
why do you add this only in this file, any reason?

Reply:
mybad no need of that line.